### PR TITLE
Add the option to contact administrator

### PR DESCRIPTION
**Pull request by federicoaldunate**
Add the option to contact the administrator.
screen shot 2018-07-03 at 2 56 33 pm
screen shot 2018-07-03 at 3 00 25 pm

### CODE REVIEW

File: app/assets/javascripts/devise.js [hunk 1]
```diff
@@ -1,5 +1,102 @@
...
+  $('.contact-success').hide();
+  $('.form-contact').show();
+  $('.contact-title-text').show();
+  $('.img-responsive').show();
+}
+
+function validateEmptyFields() {
+  let email = $.trim($('#contact-email').val());
+  let mess = $.trim($('#contact-message').val());
+  if (!(email==='' || mess ==='')) {
```
Comment (alealvarez1) [inline app/assets/javascripts/devise.js]: Add space after email

File: app/assets/stylesheets/target.sass [hunk 2]
```diff
@@ -238,3 +240,92 @@ img
...
+input.painted, textarea.painted
+    border-color: $delete-botton-color
+
+.form-contact
+  display: block
+  margin: 0 auto
+  text-align: center
+  width: 100%
+  p
+    color: $back-left-color
```
Comment (alealvarez1) [inline app/assets/stylesheets/target.sass]: Same as above

File: app/assets/javascripts/devise.js [hunk 3]
```diff
@@ -1,5 +1,102 @@
...
+  if ($('.right-triple-container').length) {
+    const widthLeft = $('.left-half-container').width();
+    $('.modal-contact').css('width',`${widthLeft-10}px`);
+    $('.modal-contact').css('min-width',`${widthLeft-10}px`);
+  } else {
+    const width = $('.modal-contact').width();
+    $('.modal-contact').css('margin-left', `-${width/2}px`);
+    $('.modal-contact').css('left', '50%');
+  }
+  const height = $('.modal-contact').height();
```
Comment (Alxzu) [inline app/assets/javascripts/devise.js]: Do you need this const?

File: app/assets/javascripts/devise.js [hunk 4]
```diff
@@ -1,5 +1,102 @@
...
+  toggleContactModal();
+}
+
+function toggleContactModal() {
+ if ($('.faded').length){
```
Comment (Alxzu) [inline app/assets/javascripts/devise.js]: Need one more space here

File: app/assets/javascripts/devise.js (hunk 1 above)
Comment (Alxzu) [inline app/assets/javascripts/devise.js]: You can change this to `if(email && mess)`

File: app/assets/javascripts/devise.js [hunk 5]
```diff
@@ -1,5 +1,102 @@
...
+}
+
+function paintRed(idField) {
+  if ($.trim($(idField).val()) === '') {
+    $(idField).addClass('painted');
+  }
+}
+
+function getDataContact() {
+  let data = {
```
Comment (Alxzu) [inline app/assets/javascripts/devise.js]: Just `return { email: $('#contact-email').val(), message: $('#contact-message').val()}`

**Pull request merged**